vllm/model_executor/layers/quantization/utils/w8a8_utils.py

```diff
@@ -121,7 +121,7 @@ def apply_fp8_linear(
         qinput, x_scale = ops.scaled_fp8_quant(
             input,
             input_scale,
-            num_token_padding=17,
+            batch_dim_padding=17,
```

Comment:
Please revert this as we decided to add quantization in the next PR.

Reply:
removed